composer.json

```diff
@@ -20,11 +20,9 @@
     "config": {
         "sort-packages": true
     },
-    "extra": {
-    },
+    "extra": {},
```

Comment:
Can be dropped

Reply:
yes. my PRs were created automatically and thus this is the result from `jq`. 

Will fix this manually when fixing the CI config file 👍🏻